beetsplug/fetchart.py

```diff
@@ -14,10 +14,15 @@
 
 """Fetches album art."""
 
+from __future__ import annotations
+
 import os
 import re
+from abc import ABC, abstractmethod
 from collections import OrderedDict
 from contextlib import closing
+from enum import Enum
+from typing import TYPE_CHECKING, AnyStr, ClassVar, Literal, Tuple, Type
```

Comment:
You might want to consider omitting AnyStr and definig the constained type yourself. `AnyStr` is getting deprecated in 3.13.